File: Lab 17 A-B Person App/B/PersonAPp/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PersonAPp
{
    class Person
    {
        public String lastName { get; private set; }
        public String firstName { get; private set; }
        private int age { get; set; }

        public Person(String firstName, String lastName, int age)
        {
            this.firstName = firstName;
            this.lastName = lastName;
            this.age = age;
        }

        public override string ToString()
        {
            return $"Person: {lastName}, {firstName}. Age: {age}";
        }
    }
}

File: Lab 17 A-B Person App/A/PersonApp/src/me/chris/Person.java
package me.chris;

public class Person {
	private String lastName;
	private String firstName;
	private int age;
	
	public Person(String firstName, String lastName, int age) {
		this.lastName = lastName;
		this.firstName = firstName;
		this.age = age;
	}
	
	public String getLastName() {
		return this.lastName;
	}
	
	

	@Override
	public String toString() {
		return "Person [lastName=" + lastName + ", firstName=" + firstName + ", age=" + age + "]";
	}
	

}
